.gitignore

```diff
@@ -53,4 +53,7 @@ types
 .jest-cache
 
 # Secrets
-secret-key
\ No newline at end of file
+secret-key
+
+# caches
+.jest-cache
```

Comment:
It is already ignored

Reply:
Ah sorry, my branch was not up-to-date, will remove.